apps/em/em-hrs-api/demo-image-policy.yaml

```diff
@@ -2,14 +2,6 @@ apiVersion: image.toolkit.fluxcd.io/v1beta1
 kind: ImagePolicy
 metadata:
   name: demo-em-hrs-api
-  annotations:
-    hmcts.github.com/prod-automated: disabled
 spec:
-  filterTags:
```

Comment:
Just comment and leave these lines. As it will be quick next time to update PR image.